README.md

```diff
@@ -43,6 +43,29 @@ For running and debugging the images:
    overlay in test/images/ instead.
  - `vm-reset`: Remove all overlays from test/images/
 
+### Creating a new image
+
+Creating a new image from scratch requires a bootstrap and a setup script,
+these scripts are located in `images/scripts`. In this example our new image is
+called `tux` and we run `./image-create -v tux` to create this image we
+expect two scripts:
+
+- `tux.bootstrap`: The bootstrap script downloads the latest available cloud image (qcow2)
```

Comment:
While most of our images start with a cloud image, some don't (cirros, bootc, rhel-7.9, rhel4edge). Perhaps

> The bootstrap script has to produce a bootable qcow2 image file, e.g. by downloading a cloud image or invoking some image build service.

Reply:
Ah! TIL